src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
```

Comment:
Don't use * selector (it impacts performance)
Better use ``html`` or ``body`` tag.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/12e17a0c-6e38-4e04-9747-a26d156207dc)